WooCommerce/Classes/ViewRelated/Dashboard/Settings/Privacy/PrivacyBannerViewController.swift

```diff
@@ -0,0 +1,115 @@
+        super.viewDidLayoutSubviews()
+        // Needed when presented via the `BottomSheetViewController`
+        preferredContentSize = .init(width: 0, height: view.intrinsicContentSize.height)
+    }
+}
+
+extension PrivacyBannerViewController: DrawerPresentable {
+    var collapsedHeight: DrawerHeight {
+        return .intrinsicHeight
+    }
+
+    var expandedHeight: DrawerHeight {
+        return .intrinsicHeight
+    }
+}
+
+/// Banner View for the privacy settings.
+///
+struct PrivacyBanner: View {
+    /// Closure to be invoked when the go to settings button is pressed.
+    ///
+    let goToSettingsAction: (() -> ())
+
+    /// Closure to be invoked when the go to settings button is pressed.
+    ///
+    let saveAction: (() -> ())
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: Layout.mainVerticalSpacing) {
+
+            Text(Localization.bannerTitle)
+                .headlineStyle()
+
+            Text(Localization.bannerSubtitle)
+                .foregroundColor(Color(.text))
+                .subheadlineStyle()
+
+            Toggle(Localization.analytics, isOn: .constant(true))
+                .tint(Color(.primary))
+                .bodyStyle()
+                .padding(.vertical)
+
+            Text(Localization.toggleSubtitle)
+                .subheadlineStyle()
+
+            HStack {
+                Button(Localization.goToSettings) {
+                    print("Tapped Settings")
+                }
+                .buttonStyle(SecondaryButtonStyle())
+
+
+                Button(Localization.save) {
+                    print("Tapped Save")
+                }
+                .buttonStyle(PrimaryButtonStyle())
+            }
+            .padding(.top)
+
+            /// Push all content to the top
+            ///
+            Spacer()
+        }
+        .padding()
+    }
+}
+
+// MARK: Definitions
+private extension PrivacyBanner {
+    enum Localization {
+        static let bannerTitle = NSLocalizedString("Manage Privacy", comment: "Title for the privacy banner")
+        static let analytics = NSLocalizedString("Analytics", comment: "Title for the analytics toggle in the privacy banner")
+        static let goToSettings = NSLocalizedString("Go to Settings", comment: "Title for the 'Go To Settings' button in the privacy banner")
+        static let save = NSLocalizedString("Save", comment: "Title for the 'Save' button in the privacy banner")
+        static let bannerSubtitle = NSLocalizedString(
+            "We process your personal data to optimize our mobile apps and marketing activities based on your consent and our legitimate interest.",
+            comment: "Title for the privacy banner"
+        )
+        static let toggleSubtitle = NSLocalizedString(
+            "These cookies allow us to optimize performance by collecting information on how users interact with our mobile apps.",
+            comment: "Description for the analytics toggle in the privacy banner"
```

Comment:
It looks like there is new copy for both of these subtitles in Figma. Should we update it now? (If we do, the longer privacy description might require some attention in landscape.)

```suggestion
        static let bannerSubtitle = NSLocalizedString(
            "Your privacy is critically important to us and always has been. " +
            "We use, store, and process your personal data to optimize our app (and your experience) in various ways. " +
            "Some uses of your data we absolutely need in order to make things work, and others you can customize from your Settings.",
            comment: "Description for the privacy banner"
        )
        static let toggleSubtitle = NSLocalizedString(
            "Allow us to optimize performance by collecting information on how users interact with our mobile apps.",
            comment: "Description for the analytics toggle in the privacy banner"
```

Reply:
Oh yeah, it looks like it got updated overnight. Will use the latest wording.

Reply:
it did create a problem on landscape, but I think the problem was always there if we considered big fonts.
I have updated the code so the banner scrolls when it grows larger than the screen height. https://github.com/woocommerce/woocommerce-ios/pull/9696/commits/1cda107a2fe422a5f50879ecad40bc91b68d18ba

Normal | Landscape
---- | ----
<img width="434" alt="Screenshot 2023-05-11 at 12 45 25 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/562080/f9f524bf-aa94-4931-86e4-e4d6e12c95b2"> | <img width="889" alt="lanscape" src="https://github.com/woocommerce/woocommerce-ios/assets/562080/fbebf606-64d5-4d67-bedf-55544282be2a">

### Big Fonts on Landscape

https://github.com/woocommerce/woocommerce-ios/assets/562080/0e76c3aa-1f33-4955-8270-9f8b81977049